<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-KCVLOK6" failures="1" tests="8" name="examplePackageForTestNg.CalculatorTest" time="0.443" errors="2" timestamp="24 Mar 2022 17:51:08 GMT">
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="add1Test" time="0.039">
    <error message="expected [10] but found [111]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [10] but found [111]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at examplePackageForTestNg.CalculatorTest.add1Test(CalculatorTest.java:13)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- add1Test -->
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="add4Test" time="0.002"/>
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="devideByZeroTest" time="0.039"/>
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="sub2Test" time="0.149"/>
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="add3Test" time="0.082"/>
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="mul1Test" time="0.077"/>
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="add2Test" time="0.017">
    <error message="expected [9] but found [11]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [9] but found [11]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at examplePackageForTestNg.CalculatorTest.add2Test(CalculatorTest.java:18)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- add2Test -->
  <testcase classname="examplePackageForTestNg.CalculatorTest" name="sub1Test" time="0.038"/>
</testsuite> <!-- examplePackageForTestNg.CalculatorTest -->
